docs/api/web-frame.md

```diff
@@ -221,4 +221,13 @@ in your app has occurred that makes you think your page is actually using less
 memory (i.e. you have navigated from a super heavy page to a mostly empty one,
 and intend to stay there).
 
+### `webFrame.setCacheCapacity(capacity)`
+
+* `capacity` Integer
+
+Attempts to set maximum size of memory cache web frame uses in bytes.
```

Comment:
I would like to avoid adding web frame since its not per frame instead per process, can be instead `Sets the capacity of resource cache for the renderer process in bytes`